Sources/tasks/Birthday-Notify.php

```diff
@@ -39,11 +39,13 @@ public function execute()
 			WHERE is_activated < 10
 				AND MONTH(birthdate) = {int:month}
 				AND DAYOFMONTH(birthdate) = {int:day}
-				AND YEAR(birthdate) > {int:year}',
+				AND YEAR(birthdate) > {int:year}
+				' . ($smcFunc['db_title'] === POSTGRE_TITLE ? 'AND indexable_month_day(birthdate) = indexable_month_day({date:bdate})' : ''),
 			array(
 				'year' => 1,
 				'month' => $month,
 				'day' => $day,
+				'bdate' => '2020-' . $month . '-' . $day,
```

Comment:
For the sake of future developers reading this, please add a comment like "// This uses 2020 merely because it is a leap year." Otherwise, this looks good.

Reply:
I will change this to 1004 like every else where we need a place holder year.